### ISSUE TITLE
Support the special key mappings as documented

### ISSUE THREAD

**Issue by dionoid** [p2000t/M2000#8]
The documentation (see below) states that there are special key mappings, e.g. for typing in "<" and ">" on modern keyboards. 
This does work in the MSDOS version, but not in the Allegro version

##Key Mappings
```
Cursor Keys, Alt/Ctrl -  Movement
Delete                -  <
Shift-Delete          -  >
` ~                   -  CODE
```

**Comment by yqxq** [comment]
My suggestion is to follow the real keyboard layout, and add this:
[ as ←
] as →
~ as ÷
| as ‖
\ as █
{ as ¼
} as ¾
^ as ↑
_ as – (long dash ascii 96)
And with alt/altg for laptops, including alt-shift support for cassette keys.
789 as numpad 789 (with shift wipe disk, ø en M)
uio as 456 (with shift INL - OPN)
jkl as 123 (with shift zoek, serial, start)
m,. as 0, 00, . (and with shift def-prnt-stop)
del as clr-screen (÷ and * have a location already)

**Comment by dionoid** [comment]
This will be picked up together with #5

**Comment by dionoid** [comment]
Fixed as part of #5
